doc/manual/17-configuration.en.md

```diff
@@ -6,14 +6,19 @@ There are two types of configuration files in Barman:
 
 - **global/general configuration**
 - **server configuration**
+- **model configuration**
 
 The main configuration file (set to `/etc/barman.conf` by default) contains general options such as main directory, system user, log file, and so on.
 
 Server configuration files, one for each server to be backed up by Barman, are located in the `/etc/barman.d` directory and must have a `.conf` suffix.
 
+Similarly, model configuration files, one for each model that can be applied to a server which is part of a cluster, are located in the `/etc/barman.d` directory and must have a `.conf` suffix.
```

Comment:
This was a little hard to parse with the extra comma separated clause - I was thinking it could be reduced to:

```suggestion
Similarly, model configuration files are located in the `/etc/barman.d` directory and must have a `.conf` suffix.
```

I was thinking of adding the `one for each model that can be applied to a server which is part of a cluster` to the end, but I wonder whether one-model-per-file is something we should recommend at all. It might be better to group multiple models for a server (or cluster) in a single file (or maybe not...?).

Reply:
I wonder if "one model per file" wouldn't end up bloating the directory 😆 I guess we can just keep the way you suggested above, and add nothing else. What do you think?

Reply:
Yeah that's fine I think.